### ISSUE TITLE
Error with Rhino CLI

### ISSUE THREAD

**Issue by nzakas** [CSSLint/csslint#189]
When I try to run the current build of the Rhino CLI, I get the following error:

org.mozilla.javascript.WrappedException: Wrapped java.util.regex.PatternSyntaxEx
ception: Unexpected internal error near index 1
\
 ^
        at org.mozilla.javascript.Context.throwAsScriptRuntimeEx(Context.java:17
57)
        at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:170)
        at org.mozilla.javascript.NativeJavaMethod.call(NativeJavaMethod.java:24
3)
        at org.mozilla.javascript.optimizer.OptRuntime.call1(OptRuntime.java:66)

```
    at org.mozilla.javascript.gen.c1._c285(Unknown Source)
    at org.mozilla.javascript.gen.c1.call(Unknown Source)
    at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:
```

97)
        at org.mozilla.javascript.gen.c1._c284(Unknown Source)
        at org.mozilla.javascript.gen.c1.call(Unknown Source)
        at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:
97)
        at org.mozilla.javascript.gen.c1._c288(Unknown Source)
        at org.mozilla.javascript.gen.c1.call(Unknown Source)
        at org.mozilla.javascript.NativeArray.iterativeMethod(NativeArray.java:1
219)
        at org.mozilla.javascript.NativeArray.execIdCall(NativeArray.java:249)
        at org.mozilla.javascript.IdFunctionObject.call(IdFunctionObject.java:12
7)
        at org.mozilla.javascript.optimizer.OptRuntime.call1(OptRuntime.java:66)

```
    at org.mozilla.javascript.gen.c1._c287(Unknown Source)
    at org.mozilla.javascript.gen.c1.call(Unknown Source)
    at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:
```

97)
        at org.mozilla.javascript.gen.c1._c277(Unknown Source)
        at org.mozilla.javascript.gen.c1.call(Unknown Source)
        at org.mozilla.javascript.optimizer.OptRuntime.callName(OptRuntime.java:
97)
        at org.mozilla.javascript.gen.c1._c0(Unknown Source)
        at org.mozilla.javascript.gen.c1.call(Unknown Source)
        at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:3
93)
        at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:283
4)
        at org.mozilla.javascript.gen.c1.call(Unknown Source)
        at org.mozilla.javascript.gen.c1.exec(Unknown Source)
        at org.mozilla.javascript.tools.shell.Main.evaluateScript(Main.java:524)

```
    at org.mozilla.javascript.tools.shell.Main.processFileSecure(Main.java:4
```

46)
        at org.mozilla.javascript.tools.shell.Main.processFile(Main.java:412)
        at org.mozilla.javascript.tools.shell.Main.processSource(Main.java:403)
        at org.mozilla.javascript.tools.shell.Main.processFiles(Main.java:179)
        at org.mozilla.javascript.tools.shell.Main$IProxy.run(Main.java:100)
        at org.mozilla.javascript.Context.call(Context.java:577)
        at org.mozilla.javascript.ContextFactory.call(ContextFactory.java:503)
        at org.mozilla.javascript.tools.shell.Main.exec(Main.java:162)
        at org.mozilla.javascript.tools.shell.Main.main(Main.java:140)
Caused by: java.util.regex.PatternSyntaxException: Unexpected internal error nea
r index 1
\
 ^
        at java.util.regex.Pattern.error(Unknown Source)
        at java.util.regex.Pattern.compile(Unknown Source)
        at java.util.regex.Pattern.<init>(Unknown Source)
        at java.util.regex.Pattern.compile(Unknown Source)
        at java.lang.String.split(Unknown Source)
        at java.lang.String.split(Unknown Source)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.mozilla.javascript.MemberBox.invoke(MemberBox.java:155)
        ... 36 more

Verified that this did not happen with the 1.6.1 release. Must be one of the changes since then.
